snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -110,7 +109,14 @@ open class Snapshot: NSObject {
         #elseif os(OSX)
             XCUIApplication().typeKey(XCUIKeyboardKeySecondaryFn, modifierFlags: [])
         #else
-            XCUIDevice.shared().orientation = .unknown
+            let screenshot = app.windows.firstMatch.screenshot()
+            guard let simulator = ProcessInfo().environment["SIMULATOR_DEVICE_NAME"], let screenshotsDir = screenshotsDirectory else { return }
+            let path = screenshotsDir.appendingPathComponent("\(simulator)-\(name).png")
+            do {
+                try screenshot.pngRepresentation.write(to: path)
+            } catch {
+                print("Problem writing screenshot: \(name) to \(path)")
```

Comment:
Can we print the actual error message here to make it easier for people to resolve the problem?